### PR TITLE
feat: reduce too many exposes module in toplevel

### PR DESCRIPTION
**Pull request by r17x**
Based on this #770

### CODE REVIEW

**Comment by davesnx** [comment]
Instead of going to that direction, probably `(wrapped false)` is a smarter move?

**Comment by r17x** [comment]
@davesnx  I'm not familiar with term of `(wrapped false)` could you explain about it?

**Comment by davesnx** [comment]
`wrapped false` is the default behaviour from ReScript where each module is exposed so given a library like reason-react you will expose all modules (React, ReactDOM, ReasonReactRouter, etc) 

With `wrapped true` you will expose this library under a name (or other more exotic forms) so lets say I name this library `React`, I could expose all modules under React now: (React.React, React.ReactDOM, React.ReasonReactRouter, etc)

**Comment by anmonteiro** [comment]
> Instead of going to that direction, probably (wrapped false) is a smarter move?

We'll need both. This PR alone still exposes all the other modules because there's `(wrapped false)` in the dune file.

File: src/dune [hunk 1]
```diff
@@ -5,5 +5,5 @@
  (public_name reason-react)
  (preprocess
   (pps melange.ppx reason-react-ppx))
- (wrapped false)
+ (wrapped true)
```
Comment (r17x) [inline src/dune:8]: I have been changes to be `true` @davesnx . I think is incomplete yet? could you give me suggestions for this changes?

Comment (anmonteiro) [reply src/dune:8]: One way to move forward with this would be to fix the build

**Comment by anmonteiro** [comment]
@davesnx do you want to include this for 0.12, or should it be included after that?

**Comment by davesnx** [comment]
Let's do it for 0.13

**Comment by anmonteiro** [comment]
This is actually going to be a lot harder than initially thought, since many modules depend on the `React` module.

**Comment by anmonteiro** [comment]
solution in #783

**Comment by r17x** [comment]
I learn a lot from #783 and I closed this PRs

Good job!

Thanks @anmonteiro

**Pull request closed without merging**